UnitTests/Plugins/DeleteUnusedBranches.Tests/GitBranchOutputCommandParserTests.cs

```diff
@@ -28,12 +28,28 @@ public void GetBranchNames_should_return_parse_correctly()
             string commandOutput = @"  feature/branch-1
 * fix/branch-2
   master
+  HEAD
+* (HEAD detached at dce3f9f)
+  +_strange_branch_name_but_valid
 
 ";
 
             _parser.GetBranchNames(commandOutput)
                    .Should()
-                   .BeEquivalentTo("feature/branch-1", "fix/branch-2", "master");
+                   .BeEquivalentTo("feature/branch-1", "fix/branch-2", "master", "+_strange_branch_name_but_valid");
+        }
+
+        [TestCase]
+        public void GetBranchNames_should_support_worktrees()
+        {
+            string commandOutput = @"+ branch_in_worktree
+*  master
```

Comment:
Should it be 2 spaces?
```suggestion
* master
```